packages/menu/src/MenuItem.tsx

```diff
@@ -86,6 +96,9 @@ const MenuItem: ExtendableBox<
     const hoverStyles = getHoverStyles(menuItemContainer.selector);
     const focusStyles = getFocusedStyles(menuItemContainer.selector);
 
+    // Darkmode is passed from the `Menu` component which is using the `useDarkMode` hook
+    const theme = darkMode ? Theme.Dark : Theme.Light;
```

Comment:
If we move to using a `MenuContext`, the `useDarkMode` hook also supports:

`const { darkMode, theme } = useDarkMode(darkModeProp);` so all children components could just consume this from `MenuContext`